ONSdigital/takeon-ui
# Feature/spp 826/adjusted value

# Description
New adjusted response column added to formview html and python script updated to retrieve new data and add it into the array for display
https://collaborate2.ons.gov.uk/jira/browse/SPP-826




# How to test?
Changes are viewable in the spp826 environment
Have included a sql insert statement in the database repo
https://github.com/ONSdigital/takeon-database/blob/feature/SPP-826/AdjustedValue/story_updates/SPP-826-Adjusted-Response-Value.sql
Only added data for reference 12345678000, script can be used in pgadmin to add responses for other responses if required
New column is observable on all forms, adjusted values will appear as a label (therefore uneditable) where data exists in the database





# Links
